plugins/cilium-cni/main.go

```diff
@@ -438,6 +438,10 @@ func cmdAdd(args *skel.CmdArgs) (err error) {
 
 	netNs, err := ns.GetNS(args.Netns)
 	if err != nil {
+		if errors.As(err, &ns.NSPathNotExistErr{}) {
```

Comment:
Let me get this straight: if we don't manage to open the netns pin from procfs, we return from `cmdAdd` successfully as if a CNI ADD was successfully completed? I don't think that's a good idea, this signals the container runtime that everything's peachy. Which problem does this patch solve, exactly?

Reply:
Sorry, Ignore this please, this is my mistake.